docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,192 @@
+
+If you have multiple commissions, depending on your setup, you will have to use priorities and groups to make sure that relevant commissions are applied. In the prior example, the merchant-specific commission with priority `1` is validated before the universal commission. If the merchant-specific commission had a lower priority, the universal commission would be applied because it doesn't have a merchant condition. For more details about priority, see []
+
+
+
+
+## Merchant commission priority and groups
+
+Priority and groups are used to make sure relevant commissions are applied when there are multiple commissions in a system.
+
+Merchant commission groups are used when multiple commissions need to be applied to per order or order item. By default, there are `primary` and `secondary` commission groups. When commissions of both groups exist in a shop, for each order or order item, one commission from each group is applied; the commission from the primary group is applied first.
+
+Merchant commission priority is used to define which commission is to be applied within a commission group when there're multiple commissions in a system. Priority is defined in an ascending order starting from 1. If groups aren't used in a system, but there're multiple commissions, priority is used to define which commission is applied in the system.
+
+For example, your setup with groups and priority could be as follows. We simplified the commissions for this example by removing irrelevant fields.
+
+| Commission Key | Priority | Group |
+| - | - | - |
+| MC01 | 1 | Primary |
+| MC02 | 2 | Primary |
+| MC03 | 2 | Secondary |
+| MC04 | 1 | Secondary |
+
+If an order item fulfills the conditions of all the commissions in the prior example, commissions `MC01` and `MC04` are applied. First, the system selects commission `MC01` as the commission in the primary group with the highest priority. Then, the system selects commission `MC04` as the commission in the secondary group with the highest priority.
+
+
+
+
+
+
+
+
+Apply commission to GROSS or NET price
+This Merchant Commission Price Mode configuration controls which prices are used as a basis to calculate commission: Gross Price or Net Price.
+
+The item Gross Price and Net Price is configured in the backend (including volume pricing) and does not include any discounts applied afterward in the shopping cart.
+
+Its defined by MERCHANT_COMMISSION_PRICE_MODE_PER_STORE, see https://github.com/spryker/spryker-docs/blob/master/_includes/pbc/all/install-features/202407.0/marketplace/install-the-marketplace-merchant-commission.md#2-set-up-configuration
+This configuration is different from an order price mode and defines behavior for the whole store.
+
```

Comment:
So, what are we trying to explain in this section?

Reply:
Marketplace Operator can choose to apply % to Gross or Net Price per store, regardless of a Price Mode selected by a customer during checkout (If Price Mode switcher feature is enabled (we are not supporting it), otherwise default price mode is used on checkout).